parsers/US_PJM.py

```diff
@@ -115,6 +119,50 @@ def data_processer(data):
     return production
 
 
+def fetch_consumption_forecast(zone_key='US-PJM', session=None,
+                               target_datetime=None, logger=None):
+    """
+    Gets consumption forecast for specified zone.
+    Returns a list of dictionaries.
+    """
+
+    if target_datetime:
+        raise NotImplementedError(
+            'This parser is not yet able to parse past dates')
+    if not session:
+        session = requests.session()
+
+    if 'PJM_TOKEN' not in os.environ:
+        raise Exception('No PJM_TOKEN found! Please add it to secrets.env!')
+    headers = {'Ocp-Apim-Subscription-Key': os.environ['PJM_TOKEN']}
+
+    # startRow must be set if forecast_area is set.
+    # RTO_COMBINED is area for whole PJM zone.
+    params = {
+        'download': True,
+        'startRow': 1,
+        'forecast_area': 'RTO_COMBINED'
+    }
+
+    # query API
+    url = API_ENDPOINT + 'load_frcstd_7_day'
+    resp = requests.get(url, params, headers=headers)
+    data = json.loads(resp.content)
+
+    data_points = list()
+    for elem in data:
+        utc_datetime = elem['forecast_datetime_beginning_utc']
+        data_point = {
+            'zoneKey': zone_key,
+            'datetime': arrow.get(utc_datetime).replace(tzinfo='UTC').datetime,
```

Comment:
It's definitely returning UTC?

Reply:
Yes :) Below is a sample data point from the API. You can see a datetime is provided for local time (EPT) and UTC time. I take the UTC datetime and set the tzinfo accordingly.
![pjm_snip](https://user-images.githubusercontent.com/6525086/41435180-ec02908e-7015-11e8-8935-e7dfe8198d2c.JPG)